lib/rules/use-ember-get-and-set.js

```diff
@@ -88,9 +82,8 @@ module.exports = {
 
     const testMethodsToSkip = new Set(['get', 'set']);
 
-    const directoriesToSkipCompletely = ['mirage'];
-
-    if (directoriesToSkipCompletely.some((dir) => isFileInDirectory(dir))) {
+    // Skip mirage directory
+    if (ember.isMirageConfig(filename)) {
```

Comment:
I thought we still want to exclude all files in the `mirage` directory from the `use-ember-get-and-set` rule? Based on what you said before, I thought we should only limit the `no-get` rule to `mirage/config.js`. If that's true, we would have two helper functions, `isMirageDirectory` and `isMirageConfig`. Correct me if I'm wrong.

Reply:
Whoops, that's right. I've corrected that now 👍